src/build.js

```diff
@@ -30,7 +34,8 @@ module.exports = function build(markdown) {
   return `
 ${doImports}
 
-module.exports = function() {
+export const attributes = ${JSON.stringify(camelize(frontMatterAttributes))};
+export default function() {
   return (
     <div>
```

Comment:
Yeah, we use `import` in the output because the output will get parsed by babel, but the module itself will not